docs/component.md

```diff
@@ -255,6 +256,50 @@ Renders:
 
     <hello-world><h1>Howdy</h1></hello-world>
 
+### Lifecycle Hooks
+
+Mainly used to set up special bindings, [can-component/connectedCallback] is defined on the viewModel and is called automatically when a component is inserted into the dom. When writing tests, since `connectedCallback` is on the viewModel, it can be called manually to to reduce complexity of tests that would otherwise need the full component to be initialized and inserted into the/a DOM. For that reason, `connectedCallback` is prefered to using `inserted` in [can-component::events events].
```

Comment:
“to to” => “to” and “ prefered” => “preferred”

I would consistently capitalize dom as DOM.